cmd/flipt/main.go

```diff
@@ -43,7 +43,28 @@ var (
 	banner       string
 )
 
-var fatal = zap.Must(zap.NewProduction()).Fatal
+var fatal = zap.Must(zap.Config{
+	Level:       zap.NewAtomicLevelAt(zap.InfoLevel),
```

Comment:
is it worth defining this `zap.Config` in a single place since its repeated in `buildConfig` also?